dependencies/supported_version/Trilinos.txt

```diff
@@ -0,0 +1 @@
+1eab15637f2998d1e86fe127b78200f2c9687cb5 2025.1 none
```

Comment:
What does the "none" mean at the end of the line?

EDIT: Ah, is this the possibility to add the version number used by Trilinos itself to later print it just for information?